swift-sdk/IterableInboxViewController.swift

```diff
@@ -0,0 +1,178 @@
+        return true
+    }
+
+    // Override to support editing the table view.
+    override open func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            // Delete the row from the data source
+            let iterableMessage = viewModels[indexPath.row].iterableMessage
+            viewModels.remove(at: indexPath.row)
+            IterableAPI.inAppManager.remove(message: iterableMessage)
+            tableView.deleteRows(at: [indexPath], with: .fade)
+        }
+    }
+    
+    override open func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let iterableMessage = viewModels[indexPath.row].iterableMessage
+        if let viewController = IterableAPI.inAppManager.createInboxMessageViewController(for: iterableMessage) {
+            IterableAPI.inAppManager.set(read: true, forMessage: iterableMessage)
+            navigationController?.pushViewController(viewController, animated: true)
+        }
+    }
+
+    private let iterableCellNibName = "IterableInboxCell"
+    
+    private var viewModels = [InboxMessageViewModel]()
+
+    private func setup() {
+        refreshMessages()
+        NotificationCenter.default.addObserver(self, selector: #selector(onInboxChanged(notification:)), name: .iterableInboxChanged, object: nil)
+    }
+    
+    deinit {
+        ITBInfo()
+        NotificationCenter.default.removeObserver(self)
+    }
+    
+    private func registerTableViewCell() {
+        let cellNibName = self.cellNibName ?? self.iterableCellNibName
+        let bundle = self.cellNibName == nil ? Bundle(for: IterableInboxViewController.self) : Bundle.main
+        
+        let nib = UINib(nibName: cellNibName, bundle: bundle)
+        tableView.register(nib, forCellReuseIdentifier: "inboxCell")
+    }
+    
+    @objc private func onInboxChanged(notification: NSNotification) {
+        ITBInfo()
+        refreshMessages()
+    }
+    
+    private func refreshMessages() {
+        DispatchQueue.main.async { [weak self] in
+            self?.viewModels.removeAll()
+            IterableAPI.inAppManager.getInboxMessages().forEach {
+                self?.viewModels.append(InboxMessageViewModel.from(message: $0))
+            }
+            
+            let unreadCount = IterableAPI.inAppManager.getUnreadInboxMessagesCount()
+            let badgeValue = unreadCount == 0 ? nil : "\(unreadCount)"
+            
+            self?.navigationController?.tabBarItem?.badgeValue = badgeValue
+            self?.tableView.reloadData()
+        }
+    }
+
+    private func configure(cell: IterableInboxCell, forViewModel viewModel: InboxMessageViewModel) {
+        cell.titleLbl?.text = viewModel.title
+        cell.subTitleLbl?.text = viewModel.subTitle
+        cell.unreadCircleView?.isHidden = viewModel.read
+        
+        cell.iconImageView?.layer.cornerRadius = 5
```

Comment:
This can be defined in the .xib file, so the customized .xib can define its own rounded corners.

Reply:
It is already defined in xib. I will remove it from here.